Added option to have input as a text file

mfoll opened the pull request:
The text file is a tab delimited file with two columns called `tumor` and `normal`with each line containing the path of the bam files.

mfoll committed:
Added option to have input as a text file

The text file is a tab delimited file with two columns called `tumor` and `normal`with each line containing the path of the bam files.

mfoll commented:
@cahaisv this is untested, please try it before I merge the branch.
@tdelhomme can you check it as well please?

tdelhomme committed:
input files from bed_tn work without suffix

tdelhomme committed:
Delete mutect.nf~

tdelhomme reviewed (approved):
Agree this is a good idea to let the user choose between provinding a file or a folder. I tested it and it needed just one change: when we get the files (tumor, normal and bed) from the 5-uplet to give to mutect, we manually wrote the name of the file using suffix, but if file provided we do not have suffixes anymore.

Commit:
Add option for all bam filepath.
Change bed extention to list
whole_genome option start to position 1 instead of 0

cahaisv reviewed (approved):
I added an option to specify the folder where all bam are located.
I corrected the extention for interval file from bed to list
I change the start position of whole genome to 1
